src/hotspot/share/runtime/globals.hpp

```diff
@@ -1304,6 +1304,12 @@ const int ObjectAlignmentInBytes = 8;
   develop(int, MaxElementPrintSize, 256,                                    \
           "maximum number of elements to print")                            \
                                                                             \
+  develop(int, MaxStringPrintSize, 256,                                     \
+          "maximum number of characters to print for a java.lang.String "   \
+          "in the VM. If exceeded, an abridged version of the string is "   \
+          "print with the middle of the string is elided.")                 \
```

Comment:
```suggestion
          "printed with the middle of the string is elided.")               \
```

Reply:
I think it should also be "... of the string elided" (without the "is").

Reply:
Fixed. Don't know how I missed that.